<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtRecorder</class>
 <widget class="QMainWindow" name="QtRecorder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>505</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>107</red>
        <green>194</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>107</red>
        <green>194</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>107</red>
        <green>194</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>107</red>
        <green>194</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>107</red>
        <green>194</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>107</red>
        <green>194</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>107</red>
        <green>194</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>107</red>
        <green>194</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>107</red>
        <green>194</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>QtRecorder</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(107, 194, 255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="nav_btn">
    <property name="geometry">
     <rect>
      <x>819</x>
      <y>411</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Next &gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="t_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>384</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Welcome to QtRecorder! </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>screencap0112</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>178</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name (without extension)</string>
    </property>
   </widget>
   <widget class="QLabel" name="display_screen">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>951</width>
      <height>481</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="d_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>591</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>In the next steps, the application will guide you to set everything up!</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>252</y>
      <width>591</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Record only a portion of the screen</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>231</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Record Audio Input</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>147</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Record Screen Input</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>611</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>START CAPTURE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pause_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>611</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>PAUSE CAPTURE </string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>611</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>STOP CAPTURE</string>
    </property>
   </widget>
   <zorder>display_screen</zorder>
   <zorder>d_label</zorder>
   <zorder>checkBox_3</zorder>
   <zorder>checkBox_2</zorder>
   <zorder>checkBox</zorder>
   <zorder>nav_btn</zorder>
   <zorder>t_label</zorder>
   <zorder>lineEdit</zorder>
   <zorder>label_3</zorder>
   <zorder>start_btn</zorder>
   <zorder>pause_btn</zorder>
   <zorder>stop_btn</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
